README.md

```diff
@@ -37,6 +37,15 @@ Extensive documentation is available [here](http://community.optimism.io/docs/).
 * [`ops`](./ops): Contains Dockerfiles for containerizing each service involved in the protocol,
 as well as a docker-compose file for bringing up local testnets easily
 
+## Contributing
+
+Read through [CONTRIBUTING.md](./CONTRIBUTING.md) for a general overview of our contribution process.
+
+### Good First Issues
+You can find good first issues by filtering for the ["good first issue" tag on our issues page](https://github.com/ethereum-optimism/optimism/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or alternatively by taking a look at our [Good First Issues project board](https://github.com/orgs/ethereum-optimism/projects/23).
```

Comment:
there are 7 open issues and 5 on the board, is that intentional? would be nice to have one as default truth

Reply:
One source of truth is better, I think we can deprecate the project board now that we have a monorepo?

Reply:
IMO both are necessary right now. We still have a few repos that are outside of the monorepo (the community hub is the main one) and my "good first issue bot" uses the project board.

Reply:
Ok thats fine as long as you are willing to maintain both